app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,33 @@
+
+<nav class="navbar navbar-expand-lg bg-light">
+  <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+
+    <% if user_signed_in? %>
+      <li class="nav-item">
+        <%= link_to 'Food List', foods_path, class: "nav-link" %>
+      </li>
+      <li class="nav-item">
+        <%= link_to 'Add Food', new_food_path, class: "nav-link" %>
+      </li>
+      <li class="nav-item">
+        <%= link_to 'Recipe List', recipes_path, class: "nav-link" %>
+      </li>
+      <li class="nav-item">
+        <%= link_to 'Public Recipe', public_recipes_path, class: "nav-link" %>
+      </li>
+      <li class="nav-item">
+        <%= link_to 'Shopping List', shopping_lists_path, class: "nav-link" %>
+      </li>
+      <li class="nav-item">
+        <strong><%= link_to current_user.name, foods_path, class: "nav-link" %></strong>
+      </li>
+    <% else %>
+      <li class="nav-item">
+        <%= link_to "Sign Up", new_user_registration_path, class: "nav-link" %>
+      </li>
+      <li class="nav-item">
+        <%= link_to "Sign In", new_user_session_path, class: "nav-link" %>
+      </li>
+    <% end %>
+  </ul>
+</nav>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you improve your design from all the pages to make them look smart and more professional. As you were provided wire **[wireframes](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md)** designed for each page, I can suggest you use them as templates.